action.yaml

```diff
@@ -62,7 +62,11 @@ runs:
 
         arch="${{ runner.arch }}"
         if [[ "${arch}" == "X64" ]]; then
-          arch="amd64"
+          if [[ "${os}" == "windows" ]]; then
+            arch="x86_64"
```

Comment:
Windows just needs to be different 😐

Reply:
Yeah, windows needs to install somewhere else too.

I'm going to ignore windows for now to get the action working again (with tests!!) and I'll fix it in a followup.